docs/architecture/adr-075-rpc-subscription.md

```diff
@@ -348,8 +348,8 @@ limit.
 
 The `wait_time` parameter is used to effect polling. If `before` is empty and
 no items are available, the server will wait for up to `wait_time` for matching
-items to arrive at the head of the log. If `wait_time` is zero or negative, the
-server will wait for a default (positive) interval.
+items to arrive at the head of the log. If `wait_time` is zero, the server will
+return whatever eligible items are available immediately.
```

Comment:
The original PR had many hunks not present here. I went through them and they are all included already  (probably because the ADRs on `main` were copied from `master`.
So all good here.